docker-compose.dev.yaml

```diff
@@ -4,7 +4,7 @@ services:
 
   marketplace-server:
     image: kinecosystem/marketplace-server:latest
-    command: ./tests/wait-for marketplace-internal:3000 -- npm run start
+    #command: ./tests/wait-for marketplace-internal:3000 -- npm run start
```

Comment:
why are these commented out?

Reply:
@nitzantomer `wait-for` didn't work on @Doc999tor 's machine (Windows). This isn't a solution though. It can be a temp fix for windows devs on their local machines until we find why wait-for failed / replace it with a different impl.

@Doc999tor remove this for the PR and leave it locally only on your machine